rni-rnt/name_groups.java

```diff
@@ -22,6 +22,7 @@ void nameGroups(String indexPathname, List<Name> names) throws NameIndexExceptio
     Name queryName = NameBuilder.data("").language(LanguageCode.UNKNOWN).extra("DEBUG").build();
     NameIndexQuery query = new NameIndexQuery(queryName);
     query.setTestNameExtra(true);
+    query.setTestEntityType(false);
```

Comment:
In the comment above this block, it's probably worth adding a line that says "The query can return results of any entity type" to clarify the change brought about by adding `testEntityType=false`